src/extension/utils/spendinggoals.js

```diff
@@ -0,0 +1,14 @@
+import { controllerLookup } from 'toolkit/extension/utils/ember';
```

Comment:
nit: rather than a new file, can we just add these functions to the bottom of your feature file